lib/nylas/model/attributes.rb

```diff
@@ -23,6 +23,7 @@ def []=(key, value)
 
       # Merges data into the registry while casting input types correctly
       def merge(new_data)
+        new_data ||= {}
```

Comment:
Is it possible to create this default assignment in the constructor? 

```ruby
def merge(new_data = {})
  # ...
end

Reply:
Removed here 36d840047b13ce05090484b4d7936e1eb1edca8f as it's not needed. Thanks